src/curl.c

```diff
@@ -363,75 +363,75 @@ enum download_status process_curl_error_codes(int curl_ret, CURL *curl_handle, i
 				response = -1; // Force error
 			}
 		}
-		debug("process_curl_error_codes: curl_ret = %d, response = %d\n", curl_ret, response);
+		debug("Curl: process_curl_error_codes: curl_ret = %d, response = %d\n", curl_ret, response);
 		/* curl command succeeded, download might've failed, let our caller handle */
 		switch (response) {
 		case 206:
-			fprintf(stderr, "Curl: Partial file downloaded from '%s'\n", url);
+			error("curl - partial file downloaded from '%s'\n", url);
 			*err = -1;
 			return DOWNLOAD_STATUS_PARTIAL_FILE;
 		case 200:
 		case 0:
 			*err = 0;
 			return DOWNLOAD_STATUS_COMPLETED;
 		case 403:
-			debug("Curl: Download failed: Forbidden (403) - '%s'\n", url);
+			debug("Curl: download failed - forbidden (403) - '%s'\n", url);
```

Comment:
there is some inconsistency in the message here too, I'd suggest using "curl - ..." over "Curl: ..."